.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ jobs:
                                   gcab
                                   ca-certificates
                                   python3-pip
-                                  mingw32-nsis
+                                  https://kojipkgs.fedoraproject.org//work/tasks/3617/86733617/msitools-0.101.32-4.fc36.x86_64.rpm
```

Comment:
drop a comment on why this specific version is needed so we know when to switch away?

Reply:
I've asked the msitools maintainer to do a new upstream release, so maybe we can just wait until the official release happens.

Reply:
I'll rip out the manual dnf install https:// when the update hits updates-testing in a few days.